cider/include/cider/CiderAllocator.h

```diff
@@ -86,12 +86,14 @@ class AlignAllocator : public CiderAllocator {
   }
 
   void deallocate(int8_t* p, size_t size) final {
-    if constexpr (ALIGNMENT > kNoAlignment && ALIGNMENT <= kMaxAlignment) {
-      uint8_t offset = *(uint8_t*)(p - 1);
-      p = adjustAddress(p, offset);
+    if (p) {
```

Comment:
better to add some log for else path, which could be issue potentially.

Reply:
OK，i will add it later

Reply:
It's ok to check here, but you should find out why p is null.

Reply:
Have already found, follow-up offline discussion